### DISCUSSION TITLE
Python 3.6.8 Pip 9.0.3 vulnerabilities not detected

### DISCUSSION THREAD

**Discussion by matjawor** [aquasecurity/trivy#5620] [False Detection]
### IDs

CVE-2022-48565,CVE-2022-37454,CVE-2021-3177,CVE-2020-27619,CVE-2019-9948,CVE-2019-9636,CVE-2019-10160

### Description

Trivy Version 0.47.0 scanned an image based on Rocky Linux 8.5 and python core packages in version below:
```
python 3.6.8
pip 9.0.3
```
Both python core packages installed by dnf package installer. Critical vulnerabilities listed in IDs section  weren't detected by Trivy image scan.

### Reproduction Steps

```bash
1. Build image based on Rocky Linux 8.5.
2. Install inside python core packages in above-mentioned versions.
- dnf install python
- dnf install pip
3. Run trivy image scanning
4. Get familiar with a report. Above-mentioned vulnerabilities will not be detected
...
```

### Target

Container Image

### Scanner

Vulnerability

### Target OS

Rocky Linux 8.5

### Debug Output

```bash
Inluded in the Description section.
```

### Version

```bash
0.47.0
```

### Checklist

- [X] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/dev/community/contribute/discussion/#false-detection)
- [X] Ran Trivy with `-f json` that shows data sources and confirmed that the security advisory in data sources was correct

**Comment by DmitriyLewen** [comment]
Hello @matjawor 
Thanks for your report!

For packages installed from package manager we use Vendor OS database - https://aquasecurity.github.io/trivy/v0.47/docs/scanner/vulnerability/#data-source-selection

It looks like [Rocky database](https://errata.rockylinux.org/) doesn't have these CVEs or there are fixes for python38 or python27.

Regards, Dmitriy
